app/editor/src/features/content/form/components/time-log/TimeLogSection.tsx

```diff
@@ -28,34 +28,27 @@ export interface ITimeLogSectionProps {
  * TimeLogSection contains the input for time tracking gor a content item.
  */
 export const TimeLogSection: React.FC<ITimeLogSectionProps> = ({ prepTimeRequired = false }) => {
-  const keycloak = useKeycloakWrapper();
-  const { values, setFieldValue } = useFormikContext<IContentForm>();
+  const { values, setFieldValue, errors } = useFormikContext<IContentForm>();
   const { isShowing, toggle } = useModal();
-  const [{ users }] = useLookup();
+  const [{ userInfo }] = useApp();
 
-  const [effort, setEffort] = React.useState(0);
   const [prep, setPrep] = React.useState<number | ''>('');
 
-  const userId = users.find((u: IUserModel) => u.username === keycloak.getUsername())?.id;
-
-  React.useEffect(() => {
```

Comment:
We don't need to manage separate state here.  We can calculate the total at any time.